### PR TITLE
supporting documentation for eclipse users using jcabi-maven-plugin

### PR DESCRIPTION
**Pull request by Agathia**
New site page for eclipse users as requested in issue #139

### CODE REVIEW

File: src/site/apt/example-weaving-eclipse.apt.vm [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  <<<pom.xml>>>, Eclipse may report the error:  
+  <<<Plugin execution not covered by lifecycle configuration: 
+  com.jcabi:jcabi-maven-plugin:0.12:ajc (execution: default, phase: 
+  process-classes)>>>
+  
+  This can be resolved by adding a new plugin to the <<<pluginManagement>>>
+  section of <<<pom.xml>>> as follows:
+
++--
+    <pluginManagement>
```
Comment (yegor256) [inline src/site/apt/example-weaving-eclipse.apt.vm]: @Agathia I think we should shift this entire block left. Now this indentation doesn't look nice :)

File: src/site/apt/example-weaving-eclipse.apt.vm [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                  </action>
+                </pluginExecution>
+              </pluginExecutions>
+            </lifecycleMappingMetadata>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
++--
+  Note that adding this section is in addition to defining the <<<plugin>>>
```
Comment (yegor256) [inline src/site/apt/example-weaving-eclipse.apt.vm]: @Agathia I would add an empty line before this paragraph

**Comment by yegor256** [comment]
@Agathia two minor comments above

**Comment by dmarkov** [comment]
Thanks for the pull request, let me find a reviewer..

**Comment by dmarkov** [comment]
@longtimeago could you please review this pull request

**Comment by longtimeago** [comment]
@yegor256 do this task really need review from my side?

**Comment by yegor256** [comment]
@longtimeago you're right, it's good as is, thanks.

**Comment by yegor256** [comment]
@rultor release, tag is `0.21.1`

**Pull request merged**